### PR TITLE
Update Croatian strings

### PR DESCRIPTION
**Pull request by anotherus3r**

### CODE REVIEW

File: locales/hr_HR/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -103,10 +103,10 @@ msgstr "Niz znakova za identifikaciju izvora."
 #. at least in:
 #. * https://duckduckgo.com/about. Appears in the window title when you open the link
 msgid "About DuckDuckGo"
-msgstr "O DuckDuckGo"
+msgstr "O DuckDuckGou"
```
Comment (jbarrett) [inline locales/hr_HR/LC_MESSAGES/duckduckgo.po]: I have a question about "DuckDuckGou". Is this change for grammatical reasons?

The string "DuckDuckGo" should be treated as a registered trademark and not translated.

**Comment by jbarrett** [comment]
Many thanks @anotherus3r!  Very sorry for the delay in reviewing.

So far, I just have one question in the comment above.

File: locales/hr_HR/LC_MESSAGES/duckduckgo.po (hunk 1 above)
Comment (anotherus3r) [reply locales/hr_HR/LC_MESSAGES/duckduckgo.po]: Yes, this is for grammatical reasons. Croatian uses cases (https://en.wikipedia.org/wiki/Serbo-Croatian_grammar#Nouns) , similar to German and Hungarian.

Comment (jbarrett) [reply locales/hr_HR/LC_MESSAGES/duckduckgo.po]: Interesting, thanks @anotherus3r. Should this apply to proper nouns also?

My thinking is a registered trade mark should be left untranslated. I am still writing the docs for this project and would like to make these cases clear and correct.

Comment (anotherus3r) [reply locales/hr_HR/LC_MESSAGES/duckduckgo.po]: Yes, this applies to proper nouns, too. Furthermore, cases also apply to adjectives, pronouns and numbers (at least in Croatian, not sure for other languages).

Comment (jbarrett) [reply locales/hr_HR/LC_MESSAGES/duckduckgo.po]: I'm leaving this comment open for now @anotherus3r. What you say makes sense, but we're going to need to develop a consistent guideline on it before we can proceed here.

Sorry for the confusion and thank you for your patience!

**Comment by jbarrett** [comment]
Many thanks @anotherus3r!

**Pull request merged**